### ISSUE TITLE
[bench request]: Add Gambler v1 (Gemma-distilled, keyless TypeSafe endpoint)

### ISSUE THREAD

**Issue by zerocorebeta** [fstandhartinger/jevbench#26]
Please evaluate **Gambler v1**, a closed Gemma-distilled decision service with native
TypeSafe wire format.

Gambler performs no text generation. It evaluates the prompt, reads only the permitted
single-token option logits at the answer position, and returns their conditional softmax.
Successful responses report `generated_tokens: 0`. The exact checkpoint, parameter count,
quantization, and serving implementation are confidential; the Gemma lineage is disclosed.

The endpoint is live and requires no API key:

```sh
python -m jevbench.cli run --tasks datasets/public/original.jsonl \
  --adapter typesafe \
  --endpoint https://gambler-relay-us-west1.leo-fish.ts.net \
  --key-env '' \
  --model gambler-v1 --delay-s 0 \
  --results RUN/results.jsonl --raw-dir RUN/raw --ledger RUN/ledger.jsonl
```

Only the public model name `gambler-v1` is accepted. The endpoint supports `noul`,
`choice`, and `score`, including string or JSON state. It is served by two independently
loaded Tesla T4 replicas behind a least-busy router; JevBench's serial run remains one
request at a time.

The probabilities are uncalibrated conditional option-marker probabilities. We performed
no JevBench-directed training or calibration and have not seen held-out items. The frozen
configuration was evaluated once on the 72 `original-public` decisions: 70/72 correct,
Brier 0.04363, ECE 0.02340. No weights, prompts, option order, or temperature were changed
from that result.

Please record the weights and serving implementation as closed and use JevBench's labelled
estimated hosted-price method for a confidential Gemma-derived size class. We can disclose
the size class privately if it is required for cost calculation.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by zerocorebeta** [comment]
Please add, demo here: https://gambler-relay-us-west1.leo-fish.ts.net/demo

Type-safe compatible, so it should be very quick to add.
